examples/cms-contentful/README.md

```diff
@@ -29,60 +29,123 @@ yarn create next-app --example cms-contentful cms-contentful-app
+After creating an account, create a new empty **space** from the [dashboard](https://app.contentful.com/) and assign to it any name of your liking.
 
-### Step 2. Create an `author` content type
+### Step 2. Create a content model
+
+The [content model](https://www.contentful.com/developers/docs/concepts/data-model/) defines the data structures of your application/websites. The structures are flexible and you can tailor them to your needs.
+
+For this example you need to create a content model that defines an author and a post content type. **You can continue creating these two by running a script or doing it manually** to familiarize yourself with the Contentful user interface.
+
+#### Run a script to creat the content model
+
+This project includes a setup script which you can use set up the content model that are expected by the source code.
+
+In your Contentful dashboard go to **Settings > General Settings** and copy the **Space ID**.
+
+Next, go to **Settings > API > Content management tokens** and create a new token by clicking **Generate personal token**. This token has the same access rights as the logged in user. **Do not share it publicly**, you will only use it to set up your space and can delete it afterwards.
+
+With the space ID and management access token at hand run the following command:
+
+```
+CONTENTFUL_SPACE_ID=YOUR_SPACE_ID CONTENTFUL_MANAGEMENT_TOKEN=XXX npm run setup
+```
+
+This command will create the needed content structures and set up your Contentful space ready to use. The output should look as follows:
+
+```
+> cms-contentful@1.0.0 setup /Users/stefan.judis/Projects/next.js/examples/cms-contentful
+> node ./contentful/setup.js $CONTENTFUL_SPACE_ID $CONTENTFUL_MANAGEMENT_TOKEN
+
+┌──────────────────────────────────────────────────┐
+│ The following entities are going to be imported: │
+├─────────────────────────────────┬────────────────┤
+│ Content Types                   │ 2              │
+├─────────────────────────────────┼────────────────┤
+│ Editor Interfaces               │ 2              │
+├─────────────────────────────────┼────────────────┤
+│ Locales                         │ 1              │
+├─────────────────────────────────┼────────────────┤
+│ Webhooks                        │ 0              │
+├─────────────────────────────────┼────────────────┤
+│ Entries                         │ 0              │
+├─────────────────────────────────┼────────────────┤
+│ Assets                          │ 0              │
+└─────────────────────────────────┴────────────────┘
+ ✔ Validating content-file
+ ✔ Initialize client (1s)
+ ✔ Checking if destination space already has any content and retrieving it (2s)
+ ✔ Apply transformations to source data (1s)
+ ✔ Push content to destination space
+   ✔ Connecting to space (1s)
+   ...
+   ...
+   ...
+```
+
+#### Create the content model manually
+
+##### Create an `Author` content type
```

Comment:
These titles are very nested and it's hard to see them as titles, maybe we can restructure things a little bit?

Reply:
The rendered view looks good to me IMO.